intranet/apps/eighth/views/admin/blocks.py

```diff
@@ -122,6 +127,7 @@ def add_block_view(request):
         "date": date,
```

Comment:
Above line 112, add `signup_time = onday[0].signup_time` to pick up the new signup_time.